<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noter</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>New file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFileExtension">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>.txt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.cpp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.hpp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.py</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.java</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.js</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.html</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.xml</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No extension</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBold">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonItalic">
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxFontSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>14</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="searchPanel" native="true">
      <property name="maximumHeight">
       <number>100</number>
      </property>
      <property name="visible">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelSearch">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSearch">
           <property name="placeholderText">
            <string>Enter text to search...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSearch">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReplace">
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReplaceAll">
           <property name="text">
            <string>Replace All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelReplace">
           <property name="text">
            <string>Replace:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditReplace">
           <property name="placeholderText">
            <string>Enter replacement text...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionToggleTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Font size</string>
   </property>
  </action>
  <action name="actionSize_2">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="actionTimes_New_Qweyke">
   <property name="text">
    <string>Times New Qweyke</string>
   </property>
  </action>
  <action name="actionQweykibri">
   <property name="text">
    <string>Qweykibri</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search &amp; Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToggleTheme">
   <property name="text">
    <string>Toggle Dark Theme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
